src/CarbonIntensity.php

```diff
@@ -368,8 +365,6 @@ public function save(string $zone_name, string $source_name, array $data): int
         );
         $stmt = $DB->prepare($query);
 
-        $in_transaction = $DB->inTransaction() || $DB->beginTransaction();
```

Comment:
Transaction handling has changed, checking if we are in a transaction is no longer possible, as methods became private.